src/js/control-bar/time-controls/time-display.js

```diff
@@ -48,7 +48,7 @@ class TimeDisplay extends Component {
       'aria-live': 'off'
     }, Dom.createEl('span', {
       className: 'vjs-control-text',
-      textContent: this.localize(this.contentText_)
+      textContent: this.localize(this.controlText_)
```

Comment:
this was a typo, I could arguable pull this into its own pr if we want, but this is one of the causes of the broken unit tests.

Reply:
This is what's causing the issue on ie8? https://travis-ci.org/videojs/video.js/jobs/290639178#L1937-L1941

Reply:
im actually not sure what is causing it